config/src/main/java/org/springframework/security/config/annotation/web/configurers/SessionManagementConfigurer.java

```diff
@@ -119,12 +120,12 @@ public final class SessionManagementConfigurer<H extends HttpSecurityBuilder<H>>
 
 	private SessionInformationExpiredStrategy expiredSessionStrategy;
 
+	private SessionLimitStrategy sessionLimitStrategy;
```

Comment:
Let's please try and align this with the reactive support and call it `SessionLimit`.